src/processor/reconnection-graph.service.ts

```diff
@@ -46,63 +61,61 @@ export class ReconnectionGraphService {
     }
 
     try {
-      // graph config and respective schema ids
-      const graphSdkConfig = await this.graphStateManager.getGraphConfig();
-
       // get the user's DSNP Graph from the blockchain and form import bundles
       // import bundles are used to import the user's DSNP Graph into the graph SDK
-      await this.importBundles(dsnpUserId, providerId, graphSdkConfig, graphKeyPairs, graphConnections, updateConnections);
-
-      let exportedUpdates: Update[] = [];
-
-      if (updateConnections) {
-        // using graphConnections form Action[] and update the user's DSNP Graph
-        const actions: ConnectAction[] = await this.formConnections(dsnpUserId, providerId, graphSdkConfig, graphConnections);
-        try {
-          await this.graphStateManager.applyActions(actions);
-        } catch (e) {
-          // silenty fail graphsdk handles duplicate connections
-          this.logger.error(`Error applying actions: ${e}`);
-        }
-        exportedUpdates = await this.graphStateManager.exportGraphUpdates();
-      } else {
-        exportedUpdates = await this.graphStateManager.forceCalculateGraphs(dsnpUserId.toString());
+      await this.importBundles(dsnpUserId, graphKeyPairs);
+      // using graphConnections form Action[] and update the user's DSNP Graph
+      const actions: ConnectAction[] = await this.formConnections(dsnpUserId, providerId, updateConnections, graphConnections);
+      try {
+        await this.graphStateManager.applyActions(actions);
+      } catch (e) {
+        // silenty fail graphsdk handles duplicate connections
+        this.logger.error(`Error applying actions: ${e}`);
```

Comment:
Can we trap the specific error here? We don't necessarily want to ignore _all_ errors.